code/modules/antagonists/changeling/powers/adrenaline.dm

```diff
@@ -19,6 +19,12 @@
 	user.SetKnockdown(0)
 	user.reagents.add_reagent("changelingadrenaline", 10)
 	user.reagents.add_reagent("changelinghaste", 2) //For a really quick burst of speed
-	user.adjustStaminaLoss(-75)
+	user.adjustStaminaLoss(-200)
```

Comment:
Changelingadrenaline already has heavy, heavy stamina regeneration. A full stamheal is straight-up overkill. -125 would likely be far less broken.

Reply:
Would something like -150 be gud?

Reply:
How'd -150 sound here?